src/lib/motion-start/components/AnimatePresence/__tests__/WaitModeFixture.svelte

```
@@ -0,0 +1,46 @@
+<script lang="ts">
+import { motion } from '../../../index.js';
+import AnimatePresence from '../AnimatePresence.svelte';
+
+let { mode = 'wait' as 'sync' | 'wait' | 'popLayout', variant = 'html' as 'html' | 'svg' | 'absolute' } = $props();
```

Biome (2.5.3)

[error] This variable mode is unused.

(lint/correctness/noUnusedVariables)